### PR TITLE
Merge gh-pages-dev into gh-pages-dev-2020.1

### PR DESCRIPTION
**Pull request by pvanderknyff**
Right now, this vnext branch does not have the latest dev changes.

### CODE REVIEW

File: docs/ui.md [hunk 1]
```diff
@@ -27,83 +27,10 @@ These elements are defined in the manifest.xml file:
 </connector-plugin>
 ```
 
-## Vendor Attributes (Custom Fields)
```
Comment (lukewrites) [inline docs/ui.md]: should this be in the 2020.1 branch?

File: docs/ui.md [hunk 2]
```diff
@@ -122,6 +49,6 @@ The `authentication-mode` and `authentication-options` tags control how a user i
 
 The other tags control what prompts show up in the connection dialog. For example, `<port-prompt value="Port: " default="5432" />` shows the Port prompt with the label of Port and a default value of 5432.
 
-## Localizing your connector
+## Localizing you connector
```
Comment (lukewrites) [inline docs/ui.md]: revert

File: docs/ui.md [hunk 3]
```diff
@@ -27,83 +27,10 @@ These elements are defined in the manifest.xml file:
...
-```
-
-
-For complete files, [Click Here](https://github.com/tableau/connector-plugin-sdk/tree/dev/samples/components/dialogs/new_text_field)
-
 ## The Tableau Custom Dialog File
 
 The UI elements you see in the dialog are determined in the .tcd file:
-```xml
+```
```
Comment (lukewrites) [inline docs/ui.md]: revert

File: docs/ui.md (hunk 1 above)
Comment (pvanderknyff) [reply docs/ui.md]: Reverted the revert, looks like everything is as expected now.

File: docs/example.md [hunk 4]
```diff
@@ -167,7 +167,7 @@ JDBCProtocol Connection URL: jdbc:postgresql://postgres:5342/TestV1?user=test&pa
 ## ![9]({{ site.baseurl }}/assets/pce-9.png) \*.tdd
 
 After connection, Tableau uses your _.tdd dialect file to determine which SQL to generate when retrieving information from your database.
-You can define your own dialect in the _.tdd file, or your connector can inherit a dialect from its parent. If you are using the 'odbc' or 'jdbc' superclasses you must 
+You can define your own dialect in the _.tdd file, or your connector can inherit a dialect from its parent. If you are using the 'odbc' or 'jdbc' superclasses you must
```
Comment (lukewrites) [inline docs/example.md]: Do we know why there's a line break here?

Comment (pvanderknyff) [reply docs/example.md]: No, fixed it

**Pull request merged**